### PR TITLE
Allow to customize keyboard shortcut from configuration file

### LINKED ISSUE #276: Allow to customize keyboard shortcut from configuration file
Might be Ctrl-Shift-S.

### PR DESCRIPTION
**Pull request by jeromerobert**
Close #276

### CODE REVIEW

File: pdfarranger/config.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    ('select(0)', '<Ctrl>a'),
+    ('select(1)', '<Ctrl><Shift>a'),
+    ('main-menu', 'F10'),
+]
+
+class Config(object):
+    """ Wrap a ConfigParser object for PDFArranger """
+
+    @staticmethod
+    def __get_action_list(m, r=[]):
```
Comment (dreua) [inline pdfarranger/config.py:51]: [Codacy actually spotted an error here](https://app.codacy.com/manual/jeromerobert/pdfarranger/file/49786423434/issues/source?bid=13283750&fileBranchId=20839044#l51): The empty list is only initialized once the class is loaded so for subsequent calls r already contains items from previous runs. Made this mistake once and it was a nightmare to figure it out. I think the best solution is `r = None` and `if not r: r = []`

Comment (jeromerobert) [reply pdfarranger/config.py]: Thanks. Fixed. I just removed the default value.

**Comment by angsch** [comment]
The GTK way of writing [platform-independent accelerators](https://mail.gnome.org/archives/commits-list/2011-September/msg06655.html) is `<Primary>` instead of `<Ctrl>`.

**Comment by jeromerobert** [comment]
@angsch Ok. This is not directly related to this PR so I'll do the change in an other commit.

**Comment by kbengs** [comment]
I realize accelerators can be set in config.py, but can it be set in a .ini file also? If so should a explanation/example be included?

**Comment by kbengs** [comment]
I saw "Unable to parse accelerator '-10': ignored request to install 1 accelerators" in terminal when I started it.

zoom-level = -10 was found under accelerators and also under DEFAULT in config.ini
After removing -10 I see no warning any more.

**Comment by jeromerobert** [comment]
Thank you for the review. Both items should now be fixed. I'll merge by the end of the week.

**Pull request merged**